tests/unit/analysis/assembly/antismash_gff_builder/test_antismash_gff_builder.yml

```diff
@@ -0,0 +1,23 @@
+- name: antismash_gff_builder test_wrong_args
+  tags:
+    - antismash_gff_builder
+  exit_code: 2
+  command: python -m mgnify_pipelines_toolkit.analysis.assembly.antismash_gff_builder --chromosome X
+  stderr:
+    contains:
+      - "antismash_gff_builder.py: error: the following arguments are required: -i/--input, -o/--output"
+- name: antismash_gff_builder test_correct_args
+  tags:
+    - antismash_gff_builder
+  command: python -m mgnify_pipelines_toolkit.analysis.assembly.antismash_gff_builder -i tests/fixtures/misc/test_antismash.json -o test.gff --cds_tag ID
+  files:
+    - path: "test.gff"
+      md5sum: 381a0c0d7e221f8817d84f36153444c4
```

Comment:
We should start uploading these files. That would make our lives easier when trying to debug issues. Otherwise, a mismatch in a digest doesn't provide much context.